### PR TITLE
Custom extensions documentation

### PR DESCRIPTION
**Pull request by dotansimha**
Plugin boilerplate available here: https://github.com/dotansimha/graphql-cli-plugin-example

### CODE REVIEW

File: CUSTOM_EXTENSION.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+## Loading GraphQL Schema
+
+To easily load GraphQL schema, you can use `loadConfig` to get it from a `graphql-config` file:
+
+```ts
+import { plugin } from '@test-graphql-cli/common';
+
+export const plugin: CliPlugin = {
+  async init({ cwd, program, loadConfig, reportError }) {
+    const config = await loadConfig();
```
Comment (wtrocki) [inline CUSTOM_EXTENSION.md]: I think using raw loadConfig returns error at the moment.

Comment (dotansimha) [reply CUSTOM_EXTENSION.md]: @kamilkisiela we must specify extensions ? no default?

**Comment by dotansimha** [comment]
@wtrocki you are right, moved to `docs`. 

@ardatan I did a fix in the code to allow sub-commands in Commander, I'm releasing a new alpha after we get this merged.

File: CUSTOM_EXTENSION.md (hunk 1 above)
Comment (wtrocki) [reply CUSTOM_EXTENSION.md]: It might be bug - not sure what is the desired api.

**Pull request merged**